packages/lib/invite/service.ts

```diff
@@ -62,27 +73,32 @@ export const deleteInvite = async (inviteId: string): Promise<TInvite> => {
   return deletedInvite;
 };
 
-export const getInviteToken = cache(async (inviteId: string) => {
-  const invite = await prisma.invite.findUnique({
-    where: {
-      id: inviteId,
-    },
-    select: {
-      email: true,
-    },
-  });
+export const getInviteToken = cache(
```

Comment:
please remove the old  react `cache` from all services :-)

Reply:
Sounds good. I was looking to implement this when I got to the caching task, but I can do the clean up here and work on actual caching in the next task